packages/client/mutations/JoinTeamMutation.ts

```diff
@@ -9,6 +9,7 @@ graphql`
     team {
       id
       name
+      isPaid
```

Comment:
-1 Why?

Reply:
If we don't include `isPaid` here, when we join a public team, the emoji suggests it's unpaid until we refresh the page:
<img width="241" alt="Screenshot 2025-03-10 at 16 49 17" src="https://github.com/user-attachments/assets/11980a25-0f40-46ae-bc25-de4087fe83bd" />